src/lib/input/input-container.ts

```diff
@@ -302,6 +336,12 @@ export class MdInputContainer implements AfterContentInit {
     this._mdInputChild._placeholderChange.subscribe(() => this._validatePlaceholders());
   }
 
+  ngAfterViewInit() {
+    // Avoid animations on load.
+    this._messageAnimationState = 'enter';
+    this._changeDetectorRef.detectChanges();
```

Comment:
@matsko can provide the definitive word here. I am pretty sure we shouldn't be calling `detectChanges` after view init just to get a certain animation behavior.